### PR TITLE
Add supports for multi-region

### LINKED ISSUE #464: Add support for multi-region
The official AWS SDK provide a "MultiRegionClient", that enables users to specify which AWS Region to send a command to by providing an `@region` input parameter

https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/s3-multiregion-client.html

We probably don't need to provide 2 Clients classes, but we can check for `@region` parameter in request.

### PR DESCRIPTION
**Pull request by jderusse**
fixes #464

### CODE REVIEW

File: src/Core/src/Sts/Input/GetCallerIdentityRequest.php [hunk 1]
```diff
@@ -6,11 +6,16 @@
 use AsyncAws\Core\Request;
 use AsyncAws\Core\Stream\StreamFactory;
 
-class GetCallerIdentityRequest implements Input
+class GetCallerIdentityRequest extends Input
 {
+    public function __construct(array $input = [])
```
Comment (Nyholm) [inline src/Core/src/Sts/Input/GetCallerIdentityRequest.php:11]: There is no doc block here. Do we care?

File: src/Core/src/Sts/StsClient.php [hunk 2]
```diff
@@ -77,7 +81,8 @@ public function assumeRoleWithWebIdentity($input): AssumeRoleWithWebIdentityResp
      */
     public function getCallerIdentity($input = []): GetCallerIdentityResponse
     {
-        $response = $this->getResponse(GetCallerIdentityRequest::create($input)->request(), new RequestContext(['operation' => 'GetCallerIdentity']));
+        $input = GetCallerIdentityRequest::create($input);
```
Comment (Nyholm) [inline src/Core/src/Sts/StsClient.php:84]: Same here, The doc block for $input does not specify `@region`.

File: src/Core/src/Sts/Input/GetCallerIdentityRequest.php (hunk 1 above)
Comment (jderusse) [reply src/Core/src/Sts/Input/GetCallerIdentityRequest.php:11]: good catch, I've to fix it

Comment (jderusse) [reply src/Core/src/Sts/Input/GetCallerIdentityRequest.php:16]: fixed

**Pull request merged**